docs/operations/rule-configuration.md

```diff
@@ -23,212 +23,242 @@ title: "Retaining or automatically dropping data"
 
-### Period Load Rule
+### Period load rule
 
-Period load rules are of the form:
+Period load rules have type `loadByPeriod` and the following example API payload:
 
 ```json
 {
-  "type" : "loadByPeriod",
-  "period" : "P1M",
-  "includeFuture" : true,
+  "type": "loadByPeriod",
+  "period": "P1M",
+  "includeFuture": true,
   "tieredReplicants": {
       "hot": 1,
-      "_default_tier" : 1
+      "_default_tier": 1
   }
 }
 ```
 
-* `type` - this should always be "loadByPeriod"
-* `period` - A JSON Object representing ISO-8601 Periods
-* `includeFuture` - A JSON Boolean indicating whether the load period should include the future. This property is optional, Default is true.
-* `tieredReplicants` - A JSON Object where the keys are the tier names and values are the number of replicas for that tier.
+Set the following properties:
+- `period`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) periods. The period is from some time in the past to the future or to the current time, depending on the `includeFuture` flag.
+- `includeFuture`: a boolean flag to indicate whether the load period includes the future. Defaults to `true`.
+- `tieredReplicants`: a JSON object containing tier names and the number of replicas for each tier.
 
-The interval of a segment will be compared against the specified period. The period is from some time in the past to the future or to the current time, which depends on `includeFuture` is true or false. The rule matches if the period *overlaps* the interval.
+Druid compares a segment's interval to the period you specify in the rule. The rule matches if the period overlaps the segment interval. 
 
-## Drop Rules
+## Drop rules
 
-Drop rules indicate when segments should be dropped from the cluster.
+Drop rules define when Druid drops segments from the cluster.
 
-### Forever Drop Rule
+### Forever drop rule
 
-Forever drop rules are of the form:
+Forever drop rules have type `dropForever`:
 
 ```json
 {
-  "type" : "dropForever"
+  "type": "dropForever"
 }
 ```
 
-* `type` - this should always be "dropForever"
+Druid drops all segments with this rule from the cluster. 
 
-All segments that match this rule are dropped from the cluster.
+### Interval drop rule
 
-
-### Interval Drop Rule
-
-Interval drop rules are of the form:
+Interval drop rules have type `dropByInterval` and the following example API payload:
 
 ```json
 {
-  "type" : "dropByInterval",
-  "interval" : "2012-01-01/2013-01-01"
+  "type": "dropByInterval",
+  "interval": "2012-01-01/2013-01-01"
 }
 ```
 
-* `type` - this should always be "dropByInterval"
-* `interval` - A JSON Object representing ISO-8601 Periods
+Set the following property:
+- `interval`: a JSON object representing [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) intervals.
 
-A segment is dropped if the interval contains the interval of the segment.
+Druid drops all segments that match the defined interval.
 
-### Period Drop Rule
+### Period drop rule
```

Comment:
Same issue with period being far more useful than a fixed interval. Consider moving this ahead of the interval drop rule.

Reply:
Done.